provider/cmd/pulumi-resource-eks/schema.json

```diff
@@ -292,10 +292,21 @@
                     "plain": true,
                     "description": "Custom k8s node labels to be attached to each worker node. Adds the given key/value pairs to the `--node-labels` kubelet argument."
                 },
+                "launchTemplateTagSpecifications": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "/aws/v6.18.2/schema.json#/types/aws:ec2%2FLaunchTemplateTagSpecification:LaunchTemplateTagSpecification"
```

Comment:
Do we have tooling that updates this reference over time? v6.18.2 I mean.

Reply:
Right now the aws provider version is centrally managed here: https://github.com/pulumi/pulumi-eks/blob/d4f539f99700f3cd084da85f3dad59ec018a2dc0/provider/cmd/pulumi-gen-eks/main.go#L82

But updating the aws version is a manual process. I'm gonna create an issue for automating that.